src/css/custom.css

```diff
@@ -31,6 +31,20 @@ html[data-theme="dark"] {
   --config-node-highlight-text-color: white;
 }
 
+.sunset-message {
+  width: 100%;
+  min-height: 5rem;
+  margin: 10px 0 20px 0;
+  padding: 15px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 8px;
+  border-left: 5px solid #fa383e;
+  background-color: rgb(75, 17, 19);
```

Comment:
![image](https://github.com/PaperMC/docs/assets/4064388/ab867056-92a4-4f83-b85d-e87daba9fd31)
Has this CSS been validated for light mode? It appears this way on Firefox and Chromium based browsers.